workflow-dependencies.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
We are not using this, right?

Reply:
This script was used to generate the list of dependencies. This is not needed after that, but it could be helpful to update dependencies in the future. Actually, it would be nice to make it update the yaml files automatically and use it to check if there is any missing dependency on each yaml file.

Would you delete it at the moment? Or do you think that there is any better folder to put it? Maybe on decidim-dev? I'd appreciate any thoughts about this 😺

Reply:
I think you can move this to the `.github/` folder, maybe even inside `.github/workflows/`, and maybe improve the docs to mention that this is used to improve the workflows files?